engine/src/flutter/shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -207,6 +207,7 @@ new PlatformMessageHandlerIos(task_runners.GetPlatformTaskRunner())) {}
   }
   [owner_controller_.platformViewsController reset];
   [owner_controller_.restorationPlugin reset];
+  [owner_controller_.textInputPlugin reset];
```

Comment:
are these `reset` only called during hot reload? Maybe rename all to `resetForHotReload`? I am pretty sure I've seen these functions but I had no idea when they are called.

Reply:
> I renamed these to onHotRestart.

nit: `resetOnHotStart` or `handleHotStart`. `onHotStart` sounds like a callback. 

> The platform view controller's reset method is called extensively throughout the iOS embedder's unit tests to reset the platform view. I kept that one as-is.

is it called in other place that's not unit test?

Reply:
Huan and I chatted offline. We decided to stick to the name "reset" with a comment that explains these are called on hot restart.

Reply:
Closing as this is addressed, but feel free to reopen if you have any follow-up feedback or questions!